lib/stdlib/src/erl_expand_records.erl

```diff
@@ -262,6 +270,42 @@ not_a_tuple({op,_,_,_}) -> true;
 not_a_tuple({op,_,_,_,_}) -> true;
 not_a_tuple(_) -> false.
 
+traverse_af(AF, Fun) ->
+    traverse_af(AF, Fun, []).
+traverse_af(AF, Fun, Acc) when is_list(AF) ->
+    [traverse_af(Ast, Fun, Fun(Ast,Acc)) || Ast <- AF];
+traverse_af(AF, Fun, Acc) when is_tuple(AF) ->
+    %% Iterate each tuple element, if the element is an AF, traverse it
+    [[(fun  (List) when is_list(List) ->
```

Comment:
A tip: there is no need create and call a fun here. You can just use a `case` or an `if` here.